File: Retos/Reto #32 - LA COLUMNA DE EXCEL [Media]/javascript/mcosme000.js
const excel = (column) => {
  const alphabet = ["A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","V","W","X","Y","Z"];
  if (column.length > 1) {
    let as_value = (column.slice(1).length) * 26
    let first_value = alphabet.indexOf(column[0]) + 1
    return first_value + as_value
  }
  return alphabet.indexOf(column) + 1
}

excel("A")
excel("Z")
excel("AA")
excel("CA")
excel("CAA")

File: Retos/Reto #32 - LA COLUMNA DE EXCEL [Media]/python/mcosme000.py
def excel(column):
    ALPHABET = ["A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","V","W","X","Y","Z"];
    if len(column) > 1:
        a_value = len(column[1:]) * 26
        first_value = ALPHABET.index(column[0]) + 1
        return a_value + first_value
    return ALPHABET.index(column) + 1

print(excel("C"))
print(excel("AAA"))
print(excel("BA"))
